bookkeeper-dist/src/main/resources/NOTICE-server.bin.txt

```diff
@@ -1,43 +1,122 @@
 Apache BookKeeper
-Copyright 2011-2017 The Apache Software Foundation
+Copyright 2011-2018 The Apache Software Foundation
 
-Licensed under the Apache License, Version 2.0 (the
-"License"); you may not use this file except in compliance
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This product contains a modified version of 'Circe', a high-performance
+hash algorithm framework & library from Trevor Robinson:
+
+  * LICENSE: Apache License 2.0
```

Comment:
I don't think we need any copy rights in NOTICE per Justin MClean. All these below are already in LICENSE. NOTICE is informative and should be as brief as possible.

Reply:
done